src/main/java/com/mineboundteam/minebound/item/ToolTier.java

```diff
@@ -0,0 +1,12 @@
+package com.mineboundteam.minebound.item;
+
+import com.mineboundteam.minebound.registry.ItemRegistry;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraftforge.common.ForgeTier;
+
+public class ToolTier {
+    public static final ForgeTier ENERGIZED_IRON = new ForgeTier(2, 250, 7, 2, 8, BlockTags.NEEDS_IRON_TOOL, () -> Ingredient.of(ItemRegistry.ENERGIZED_IRON_INGOT.get()));
+    public static final ForgeTier ILLUMINANT_STEEL = new ForgeTier(2, 1561, 7.5F, 2.3F, 8, BlockTags.NEEDS_IRON_TOOL, () -> Ingredient.of(ItemRegistry.ILLUMINANT_STEEL_INGOT.get()));
+    public static final ForgeTier GILDED_DIAMOND = new ForgeTier(3, 1561, 8, 3, 8, BlockTags.NEEDS_DIAMOND_TOOL, () -> Ingredient.of(ItemRegistry.GILDED_DIAMOND_INGOT.get()));
```

Comment:
Boost the durability a bit; maybe around 1700. Enchantability can be around 15-20.

Reply:
I also realized something about the mining speed: speed is reduced when the block can't break it. Instead of searching for exactly stone (which isn't correct because it would also need to consider _stones_ like diorite, cobble, deepslate, etc) maybe make the base speed higher, like 10.5 or 11